<ui version="4.0" >
 <class>HistoricCfgWidget</class>
 <widget class="QWidget" name="HistoricCfgWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>220</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>220</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Historic Configuration Tool&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLineEdit" name="txtName" >
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="icepapcms.qrc" >:/small_icons/IcepapCfg Icons/redpixel.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Description</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QTextEdit" name="txtDescription" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_calendar" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QCalendarWidget" name="calendarWidget" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="firstDayOfWeek" >
          <enum>Qt::Monday</enum>
         </property>
         <property name="gridVisible" >
          <bool>true</bool>
         </property>
         <property name="verticalHeaderFormat" >
          <enum>QCalendarWidget::NoVerticalHeader</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_day" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" colspan="2" >
        <widget class="QListWidget" name="listWidget" />
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QPushButton" name="btnBack" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icepapcms.qrc" >:/small_icons/IcepapCFG Icons Petits/go-relative2.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveButton" >
       <property name="text" >
        <string>Save</string>
       </property>
       <property name="icon" >
        <iconset resource="icepapcms.qrc" >:/small_icons/IcepapCFG Icons Petits/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton" >
       <property name="text" >
        <string>Delete</string>
       </property>
       <property name="icon" >
        <iconset resource="icepapcms.qrc" >:/small_icons/IcepapCFG Icons Petits/process-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icepapcms.qrc" />
 </resources>
 <connections/>
</ui>
